src/entities/Employee.java

```diff
@@ -0,0 +1,21 @@
+package entities;
+
+public class Employee extends User{
```

Comment:
Could you implement equals and hashcode?

Also you can add another commit to this pr to change it on user since it is already merged in the dev branch